doc/source/ray-overview/learn-more.md

```diff
@@ -1,8 +1,25 @@
 # Learn More
 
-Here are some talks, papers, and press coverage involving Ray and its libraries.
+Here are some general rules of thumb, talks, papers, and press coverage involving Ray and its libraries.
 Please raise an issue if any of the below links are broken, or if you'd like to add your own talk!
 
+## Latency numbers every Ray programmer should know
+
+| Stat | Value |
+| --- | --- | 
+| Typical Ray cluster startup time | ~6 minutes |
+| Max size of Ray cluster | ~500 machines |
+| Typical base image size (compressed) | ~4GB | 
+| Typical base image size for GPU (compressed) | ~9GB | 
+| Max number of actors | ~10,000 |
+| Task latency in Ray (add 2 numbers) | ~10ms |
+| Tasks processed per second (add 2 numbers | ~20,000/second |
+| Minimal memory footprint of a task (Python) | ~200MB |
```

Comment:
it's worth noting this number is dominated by ray's deps and python runtime where incremental memory footprint of a task / actor is much, much smaller. It looks a bit scary when i read it the first time ..

Reply:
Yeah maybe saying "Python worker" instead of "a task" is more accurate.

Reply:
+1 -- updated to specify `Python worker` and `Python actor`. 👍